geth-patches/README.md

```diff
@@ -59,25 +51,23 @@ Once patches applied, you might want to inspect changes between current vendored
 diff -Nru -x "*_test.go" -x "vendor" -x ".git" -x "tests" -x "build" --brief $GOPATH/src/github.com/status-im/go-ethereum $GOPATH/src/github.com/status-im/status-go/vendor/github.com/ethereum/go-ethereum
 ```
 
-# Vendor patched version
-## Using `dep` tool
-
-TBD
+4. Push `develop` branch to our remote, rewriting history
+```
+git push -f origin develop
+```
 
-## Manually
-This method should be used only while `dep` tool workflow is not set up.
+#### II. In `status-go` repository
```

Comment:
Also here status-go is a name, no code. So simply

#### II. In status-go repository